lms/djangoapps/course_api/tests/test_serializers.py

```diff
@@ -164,6 +164,7 @@ def setUp(self):
         # update the expected_data to include the 'overview' data.
         about_descriptor = XBlock.load_class('about')
         overview_template = about_descriptor.get_template('overview.yaml')
+        self.expected_data['certificate_available_date'] = '2015-08-14T00:00:00Z'
```

Comment:
I think we should also add a test where this is `None` when the certificate display behavior is not turned on.

Reply:
This check is not on the `serializers.py` so I can't add it in `test_serializers.py` because the serializer will return according to the received value if it's `None` it'll return `None` else it'll return a `DateTime` value. I've added the `ENABLE_V2_CERT_DISPLAY_SETTINGS` check in [api.py:course_detail](https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/course_api/api.py#L56) So the test can be added here in [test_api.py:TestGetCourseDetail](https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/course_api/tests/test_api.py#L65). But I've explored the tests here and I've not found any test case to check response fields. I think we don't need a test case for `None`

@pdpinch What are your thoughts on this? As @arslanashraf7 is on leaves